server/models/video.js

```diff
@@ -303,6 +303,12 @@ function afterDestroy (video, options, next) {
         return callback()
       }
     )
+  } else {
```

Comment:
You don't need this because we put a `onDelete: 'cascade'` on the `Videos`/`BlacklistedVideos` relation.